tools/registry.py

```diff
@@ -269,31 +269,38 @@ def add(self, module, override=False):
       shutil.copy(module.presubmit_yml, presubmit_yml)
     else:
       PLATFORMS = ["centos7", "debian10", "ubuntu2004", "macos", "windows"]
-      presubmit = dict()
-      presubmit["matrix"] = dict()
-      presubmit["matrix"]["platform"] = PLATFORMS.copy()
-      presubmit["tasks"] = dict()
-      task = dict()
-      task["name"] = "Verify build targets"
-      task["platform"] = "${{ platform }}"
-      if module.build_targets:
-        task["build_targets"] = module.build_targets.copy()
-      presubmit["tasks"]["verify_targets"] = task
+      task = {
+        "name": "Verify build targets",
+        "platform": "${{ platform }}",
+        "build_targets": module.build_targets.copy()
```

Comment:
can `module.build_targets` be None? if not, we could just merge the whole `task` literal inside `presubmit`

Reply:
It won't be None, yeah indeed we could!